src/bus/socket.spec.js

```diff
@@ -0,0 +1,59 @@
+import { WebSocket } from 'mock-socket';
+import Socket from './socket';
+
+describe('Socket', function() {
+  let socketUrl;
+  let webSocket;
+
+  beforeEach(function() {
+    this.sandbox = sandbox.create();
+
+    socketUrl = `wss://${faker.internet.domainName()}`;
+    webSocket = new WebSocket(socketUrl);
+  });
+
+  afterEach(function() {
+    this.sandbox.restore();
+
+    webSocket.close();
+  });
+
+  describe('constructor', function() {
+    let expectedOrganization;
+    let socket;
+
+    beforeEach(function() {
+      expectedOrganization = fixture.build('organization');
+      socket = new Socket(webSocket, expectedOrganization.id);
+    });
+
+    it('sets a socket for the class instance', function() {
+      expect(socket._socket).to.deep.equal(webSocket);
+    });
+
+    it('sets an organization id for the class instance', function() {
+      expect(socket._organizationId).to.equal(expectedOrganization.id);
+    });
+
+    it("returns a function called 'close'", function() {
+      expect(socket.close).to.be.a('function');
+    });
+  });
+
+  describe('close', function() {
+    let close;
+    let expectedOrganization;
+    let socket;
+
+    beforeEach(function() {
+      close = this.sandbox.stub(webSocket, 'close');
+      expectedOrganization = fixture.build('organization');
+      socket = new Socket(webSocket, expectedOrganization.id);
+      socket.close();
```

Comment:
Could you change this slightly to match the style we've been using elsewhere:
1.build the fixtures
2. blank line
3. stub stuff
4. blank line
5. actually run it (and create the instance)